src/hooks/queries/useHomeGalleryAssets.js

```diff
@@ -38,7 +39,55 @@ const useHomeGalleryAssets = () => {
     }
   `);
 
+  let LARGE_PICTURE = {};
+  let MEDIUM_PICTURE = {};
+  let SMALL_PICTURE = {};
+
+  const { breakpoint, width } = useBreakpoints({
+    breakpoints: {
+      desktopLg: 1280,
+      desktop: 768,
+    },
+    debounceDelay: 250,
+  });
+
+  if (breakpoint === 'desktop') {
+    LARGE_PICTURE.WIDTH = '400px';
+    LARGE_PICTURE.HEIGHT = '280px';
+    MEDIUM_PICTURE.WIDTH = '300px';
+    MEDIUM_PICTURE.HEIGHT = '220px';
+    SMALL_PICTURE.WIDTH = '240px';
+    SMALL_PICTURE.HEIGHT = '180px';
+  } else if (breakpoint === 'desktopLg') {
```

Comment:
since we added mobile sizes as defaults, we can remove `else if` statement and keep only
```
if (breakpoint === 'desktop') {
    LARGE_PICTURE.WIDTH = '400px';
    LARGE_PICTURE.HEIGHT = '280px';
    MEDIUM_PICTURE.WIDTH = '300px';
    MEDIUM_PICTURE.HEIGHT = '220px';
    SMALL_PICTURE.WIDTH = '240px';
    SMALL_PICTURE.HEIGHT = '180px';
  }
```